src/config/mod.rs

```diff
@@ -28,16 +28,21 @@ pub const DEFAULT_PROF_NAME: &str = "default";
  All should start with ENV_VAR_PREFIX (CT_xxx).
 ************************************************************************/
 // Default prefix for environment variables added by CloudTruth.
-pub const ENV_VAR_PREFIX: &str = "CT_";
+pub const ENV_VAR_PREFIX: &str = "CLOUDTRUTH_";
 
 // Environment variable name used to specify the CloudTruth API value, so it does not need to be
 // specified on the command line.
 #[allow(dead_code)]
-pub const CT_API_KEY: &str = "CT_API_KEY";
+pub const CT_API_KEY: &str = "CLOUDTRUTH_TOKEN";
+
+// The old environment variable was named 'CT_API_KEY', and we want to provide a better
```

Comment:
Isn't there also a CT_ENV, or was that not in the CLI yet?

Reply:
The CT_ENV has not been moved into a proper environment variable yet. It is dynamically created only in the `cloudtruth run` command.

Reply:
It does use the ENV_VAR_PREFIX, so it will be adjusted with these changes.